# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295104803 28800
#      Sat Jan 15 07:20:03 2011 -0800
# Node ID a34ccc8e2f8e83cee0896b9cd11ab5369e2dbc40
# Parent  8f90ffe3c5f994c20ba603433db7d586a9a991a0
Update list of exceptions not requiring documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-15  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/doccheck/mk_undocumented_list: Update list of exceptions
+	not requiring documentation.
+
 2011-01-15  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/stats.txi: Add stdnormal_xxx functions to documentation.
 
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/matrix.txi: Add undocumented postpad to documentation.
 
diff --git a/doc/interpreter/doccheck/mk_undocumented_list b/doc/interpreter/doccheck/mk_undocumented_list
--- a/doc/interpreter/doccheck/mk_undocumented_list
+++ b/doc/interpreter/doccheck/mk_undocumented_list
@@ -72,52 +72,56 @@ exit(1);
 ################################################################################
 __DATA__
 angle
 bessel
 besselh
 besseli
 besselk
 bessely
+bug_report
 chdir
 comma
 debug
 exit
 F_DUPFD
 F_GETFD
 F_GETFL
 finite
 fntests
 F_SETFD
 F_SETFL
 gammaln
+home
 i
 inf
 inverse
 isbool
 isfinite
 j
 J
 lower
+lstat
 nan
 O_APPEND
 O_ASYNC
 O_CREAT
 O_EXCL
 O_NONBLOCK
 O_RDONLY
 O_RDWR
 O_SYNC
 O_TRUNC
 O_WRONLY
 paren
 SEEK_CUR
 SEEK_END
 semicolon
 setenv
+shell_cmd
 toc
 triu
 unimplemented
 upper
 ylabel
 ylim
 zlabel
 zlim
